main.tf

```diff
@@ -1,3 +1,336 @@
-/********************************************************************
-This file is used to implement the ROOT module.
-*********************************************************************/
+##############################################################################
+# External Secrets Sync Module
+#
+# Module for deploying External Secret Operator (ESO) and use it to create and synchronize Kubernetes secrets into clusters based on Secrets-Manager secrets.
+##############################################################################
+
+## Install ESO
+
+locals {
+  eso_image_tag_digest = "v0.10.5-ubi@sha256:2d4583ff06dd9e186076cc11f2c4782f17e32da28fe18b1144d89806213d17cd" # datasource: icr.io/ibm-iac/external-secrets
+  eso_image_repo       = "icr.io/ibm-iac/external-secrets"
+
+  reloader_image_tag_digest = "v1.1.0-ubi@sha256:d2adccbac4da35a8b31aa21bfbc44da47d8676a41b91fd8f3b192f386f72aa20" # datasource: icr.io/ibm-iac/reloader
+  reloader_image_repo       = "icr.io/ibm-iac/reloader"
```

Comment:
As per internal conversations, we should change this to pull from the original source registry (which is ghcr.io), but provide the ability to overwrite this in a new optional variable

Reply:
@ocofaigh , implementing it.